Jenkinsfile

```diff
@@ -48,30 +53,50 @@ pipeline {
         }
         stage('Parallel tests') {
             parallel {
+                
                 stage('Windows interface tests') {
                     agent { label 'windows' }
                     steps {
                         setupCXX()
                         bat runTests()
                     }
-                    post { always { deleteDir() }}
+                    post { 
+                        always { 
+
+                            recordIssues id: "Windows",
+                            name: "Windows interface tests",
+                            enabledForFailure: true,
+                            aggregatingResults : true,
+                            tools: [
+                                gcc4(id: "Windows_gcc4", name: "Windows interface tests@GCC4"),
```

Comment:
This `id` thing is very cool, did not know how to do that before and we got kind of messy results from it. Thanks :D